riotfile.py

```diff
@@ -1416,6 +1416,62 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ],
             },
         ),
+        Venv(
+            name="sanic",
+            command="pytest {cmdargs} tests/contrib/sanic",
+            pkgs={
+                "pytest-asyncio": latest,
+                "requests": latest,
+            },
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version="3.7", max_version="3.9"),
+                    pkgs={
+                        "sanic": ["~=19.12"],
+                        "pytest-sanic": ["~=1.6.2"],
+                        "httpx": ["~=0.9.3"],
```

Comment:
I don't see httpx used now with a server starting with `run_server.py` so we can safely remove it.

Reply:
I'll leave a comment, this is needed because otherwise pytest-sanic will try to install a newer version which is incompatible

Reply:
```suggestion
                        # Pin the version of httpx because pytest-sanic will try to install a newer/incompatible version
                        "httpx": ["~=0.9.3"],
```

Reply:
... I think... now you have me appropriately second guessing. I will check.

Reply:
good catch, these isn't needed. updated.